apps/frontend/src/libs/components/balance-wheel-chart/libs/helpers/generate-gradient.helper.ts

```diff
@@ -0,0 +1,53 @@
+import { type ScriptableContext } from "~/libs/types/types.js";
+
+import { FALLBACK_BACKGROUND_COLOR } from "../constants/constants.js";
+import { CHART_SLICE_COLORS } from "../enums/enums.js";
+
+const GRADIENT_SETTING = {
```

Comment:
```suggestion
const GradientSetting = {
```
Check B23 rule for enum naming